module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/New-EntraBetaApplicationPasswordCredential.md

```diff
@@ -55,23 +57,35 @@ This command creates new password credential for specified application.
 ### Example 2: Create a password credential using CustomKeyIdentifier parameter
 
 ```powershell
-New-EntraBetaApplicationPasswordCredential -ObjectId 'tttttttt-0000-2222-0000-aaaaaaaaaaaa' -CustomKeyIdentifier 'demoPassword'
+Connect-Entra -Scopes 'Application.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Application.ReadWrite.OwnedBy' #Application Permission
+$params = @{
+    ObjectId = 'tttttttt-0000-2222-0000-aaaaaaaaaaaa'
+    CustomKeyIdentifier = 'demoPassword'
+}
+
+New-EntraBetaApplicationPasswordCredential @params
 ```
 
 ```output
-CustomKeyIdentifier                           DisplayName  EndDateTime          Hint KeyId                                SecretText                               StartDat
-                                                                                                                                                                   eTime
--------------------                           -----------  -----------          ---- -----                                ----------                               --------
-100 101 109 111 80 97 115 115 119 111 114 100 demoPassword 6/10/2026 7:43:45 AM 9tb  tttttttt-0000-2222-0000-aaaaaaaaaaaa wbBNW8kCuiPjNRg9NX98W_EaU6cqG 6/10/...
-
+CustomKeyIdentifier DisplayName EndDateTime          Hint KeyId                                SecretText                    StartDateTime
+------------------- ----------- -----------          ---- -----                                ----------                    -------------
+                                3/21/2026 9:48:40 AM n34  tttttttt-0000-2222-0000-aaaaaaaaaaaa wbBNW8kCuiPjNRg9NX98W_aaaaaaa 3/21/2024 9:48:40 AM
 ```
 
 This command creates new password credential for specified application.
 
 ### Example 3: Create a password credential using StartDate parameter
 
 ```powershell
-New-EntraBetaApplicationPasswordCredential -ObjectId 'tttttttt-0000-2222-0000-aaaaaaaaaaaa' -StartDate (get-date).AddYears(0)
+Connect-Entra -Scopes 'Application.ReadWrite.All' #Delegated Permission
+Connect-Entra -Scopes 'Application.ReadWrite.OwnedBy' #Application Permission
+$params = @{
+    ObjectId = 'tttttttt-0000-2222-0000-aaaaaaaaaaaa'
+    StartDate = (Get-Date).AddYears(0)
+}
+
+New-EntraBetaApplicationPasswordCredential @params
```

Comment:
In example description we need t to add used parameters description for ObjectId and StartDate